var/spack/repos/builtin/packages/py-jupyter-core/package.py

```diff
@@ -26,5 +27,7 @@ class PyJupyterCore(PythonPackage):
     version('4.0.0', sha256='9025208cdfc40718c7e3ab62b5e17aacf68e3fc66e34ff21fe032d553620122a')
 
     depends_on('python@2.7:2.8,3.3:', type=('build', 'run'))
+    depends_on('python@2.7:2.8,3.5:', type=('build', 'run'), when='@4.6.2:')
```

Comment:
I don't think this is the case, from the `setup.cfg`:
```
python_requires = >=2.7, !=3.0, !=3.1, !=3.2
```

Reply:
https://github.com/jupyter/jupyter_core/blob/58b710c0ec711bc7789bd3e9fe52cc2a6eba6957/setup.cfg#L25

Reply:
Ah sorry, I was looking at 4.6.0, not 4.6.3.